charts/envoy/templates/psp-clusterrolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if and ( .Values.podSecurityPolicy.enabled ) ( .Values.rbac.create ) }}
```

Comment:
Role for using PodSecurityPolicy by linking it to the service account used for deployment.